dev-client/src/screens/SoilScreen/components/EditIntervalModalContent.tsx

```diff
@@ -84,13 +104,24 @@ export const EditIntervalModalContent = ({siteId, depthInterval}: Props) => {
     [soilData],
   );
 
+  const updateSwitch = useCallback(
+    (method: SoilPitMethod) => (newValue: boolean) => {
+      dispatch(
+        updateSoilDataDepthIntervalAsync({
+          siteId,
+          depthInterval,
+          [methodEnabled(method)]: newValue,
+        }),
+      );
+    },
+    [dispatch, depthInterval, siteId],
+  );
+
   const onSubmit = useCallback(
     async (values: EditIntervalFormInput) => {
-      // TODO: actually use the applyToAll variable
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      const {start, end, applyToAll, ...newInterval} = schema.cast(values);
+      const {start, end, ...newInterval} = schema.cast(values);
```

Comment:
housekeeping: please either restore the TODO comment or file a follow-up issue to handle the "apply to all" switch!

Reply:
Think I did it in a following PR